lib/toTree.js

```diff
@@ -21,8 +20,7 @@ function innerToTree(seed, path, depth) {
     key = iterateKeySet(keySet, iteratorNote);
 
     while (!iteratorNote.done) {
-
-        var next = seed[key];
+        var next = Object.hasOwnProperty.call(seed, key) && seed[key];
```

Comment:
I would recommend assigning `Object.prototype.hasOwnProperty` to a top-level variable and using that.